test/verify/check-storage-lvm2

```diff
@@ -450,6 +450,8 @@ class TestStorageLvm2(storagelib.StorageCase):
         b.wait_js_func("ph_count_check", card + f" .storage-stripe-pv-box-dev:contains('{bn(disk3)}')", 1)
         b.wait_js_func("ph_count_check", card + f" .storage-stripe-pv-box-dev:contains('{bn(disk4)}')", 1)
         b.wait_js_func("ph_count_check", card + f" .storage-stripe-pv-box-dev:contains('{bn(disk1)}')", 1)
+
+        b.wait(lambda: float(self.machine.execute("lvs vgroup0/lvol0 -o sync_percent --noheading")) == 100)
```

Comment:
Shouldn't this wait for the rendered percentage display instead? There may still be some lag between what the OS thinks, and what bubbled through udisks/D-Bus/bridge/channel/React?